packages/mdc-typography/_mixins.scss

```diff
@@ -35,6 +35,10 @@
   font-weight: #{map-get($style-props, font-weight)};
   letter-spacing: map-get($style-props, letter-spacing);
   line-height: map-get($style-props, line-height);
+
+  @if $style == "button" {
```

Comment:
There is a line on spec said:

> “Button style (Medium 14sp, all caps) is used for all buttons. Button text should be all caps in languages that have capitalization. ”

So I think we should enforce capitalization but as you suggested not `text-align` center, etc.., SG?There is a line on spec said:

> “Button style (Medium 14sp, all caps) is used for all buttons. Button text should be all caps in languages that have capitalization. ”

So I think we should enforce capitalization but as you suggested not `text-align` center, etc.., SG?